menu/menu.py

```diff
@@ -0,0 +1,58 @@
+order = []
+price = []
+
+menu = {
+    "Baja Taco": 4.00,
+    "Burrito": 7.50,
+    "Bowl": 8.50,
+    "Nachos": 11.00,
+    "Quesadilla": 8.50,
+    "Super Burrito": 8.50,
+    "Super Quesadilla": 9.50,
+    "Taco": 3.00,
+    "Tortilla Salad": 8.00
+}
+menu =  {k.lower(): v for k, v in menu.items()}
+
+def taking_order(order, price, menu):
+    while True:
+        dish = input('Type food from menu which you want to order: ').lower()
```

Comment:
You can use [titlecase()](https://docs.python.org/3/library/stdtypes.html#str.title) and get rid of line 15